app/docs/2.0.x/explore/dpp-on-kubernetes.md

```diff
@@ -164,42 +164,44 @@ Notice how `kuma.io/service` is built on `<serviceName>_<namespace>_svc_<port>`
 
 ### Joining the mesh
 
-On Kubernetes, `Dataplane` resource is automatically created by kuma-cp. For each Pod with sidecar-injection label a new
+On Kubernetes, `Dataplane` resource is automatically created by kuma-cp. For each `Pod` with sidecar-injection label, a new
 `Dataplane` resource will be created.
 
 To join the mesh in a graceful way, we need to first make sure the application is ready to serve traffic before it can be considered a valid traffic destination.
 
-When Pod is converted to a `Dataplane` object it will be marked as unhealthy until Kubernetes considers all containers to be ready.
+When `Pod` is converted to a `Dataplane` object it will be marked as unhealthy until Kubernetes considers all containers to be ready.
 
 ### Leaving the mesh
 
 To leave the mesh in a graceful shutdown, we need to remove the traffic destination from all the clients before shutting it down.
 
-When the {{site.mesh_product_name}} DP sidecar receives a `SIGTERM` signal it does in this order:
+When the {{site.mesh_product_name}} sidecar receives a SIGTERM signal it:
 
-1) start draining Envoy listeners
-2) wait for the entire draining time
-3) stop the sidecar process
-During the draining process, Envoy can still accept connections however:
-1) It is marked as unhealthy on Envoy Admin `/ready` endpoint
-2) It sends `connection: close` for HTTP/1.1 requests and GOAWAY frame for HTTP/2.
-   This forces clients to close a connection and reconnect to the new instance.
+1. Starts draining Envoy listeners.
```

Comment:
Numbers aren't increasing

Reply:
This is markdown, it makes you not have to ever renumber them if anything changes. It's rendered properly

Reply:
I suppose it might be something we want to agree on if it bothers anyone in the raw markdown